tests/data/power_flow/1os2msr/input.json

```diff
@@ -1,69 +1,75 @@
 {
-  "node": [
-    {
-      "id": 1,
-      "u_rated": 10.5e3
-    },
-    {
-      "id": 2,
-      "u_rated": 10.5e3
-    },
-    {
-      "id": 3,
-      "u_rated": 10.5e3
-    }
-  ],
-  "line": [
-    {
-      "id": 4,
-      "from_node": 1,
-      "to_node": 2,
-      "from_status": 1,
-      "to_status": 1,
-      "r1": 0.11,
-      "x1": 0.12,
-      "c1": 4.1380285203892784e-05,
-      "tan1": 0.1076923076923077,
-      "i_n": 510.0
-    },
-    {
-      "id": 5,
-      "from_node": 2,
-      "to_node": 3,
-      "from_status": 1,
-      "to_status": 1,
-      "r1": 0.15,
-      "x1": 0.16,
-      "c1": 5.411268065124442e-05,
-      "tan1": 0.10588235294117646,
-      "i_n": 520.0
-    }
-  ],
-  "source": [
-    {
-      "id": 6,
-      "node": 1,
-      "status": 1,
-      "u_ref": 1.01047619047619,
-      "sk": 1e20
-    }
-  ],
-  "sym_load": [
-    {
-      "id": 7,
-      "node": 2,
-      "status": 1,
-      "type": 0,
-      "p_specified": 1.01e6,
-      "q_specified": 0.21e6
-    },
-    {
-      "id": 8,
-      "node": 3,
-      "status": 1,
-      "type": 0,
-      "p_specified": 1.02e6,
-      "q_specified": 0.22e6
-    }
-  ]
+  "attributes": {},
+  "data": {
+    "line": [
+      {
+        "c1": 4.1380285203892784e-05,
+        "from_node": 1,
+        "from_status": 1,
+        "i_n": 510,
+        "id": 4,
+        "r1": 0.11,
+        "tan1": 0.1076923076923077,
+        "to_node": 2,
+        "to_status": 1,
+        "x1": 0.12
+      },
+      {
+        "c1": 5.411268065124442e-05,
+        "from_node": 2,
+        "from_status": 1,
+        "i_n": 520,
+        "id": 5,
+        "r1": 0.15,
+        "tan1": 0.10588235294117646,
+        "to_node": 3,
+        "to_status": 1,
+        "x1": 0.16
+      }
+    ],
+    "node": [
+      {
+        "id": 1,
+        "u_rated": 10500
```

Comment:
Used to be `10.5e3`, which seems more readable to me (especially for other values like `p_specified` -> see below)

Reply:
not sure we can do that unfortunately... `msgpack` decides for us. if you think it's important, i will have to manually go over all test files or write a separate script that does this

Reply:
We can write a simple script to do string concatenation to transform existing json files.

Reply:
yes that's what i'm doing now after a quick discussion with peter